peepholelib/classifier/classifier_base.py

```diff
@@ -78,7 +97,8 @@ def compute_empirical_posteriors(self, **kwargs):
         for batch in tqdm(self._fit_dl, disable=not verbose):
             data = self.parser(data=batch, **self.parser_kwargs).to(self.device)
             preds = self._classifier.predict(data)
-            labels = batch['label']
+            # labels = batch['label']
```

Comment:
Here we have two parsers, it would be best to have just one that does the job of both.